app/models/appointment.rb

```diff
@@ -1,5 +1,34 @@
 class Appointment < ActiveRecord::Base
 	belongs_to :doctor
 	belongs_to :patient
-	validates_presence_of :doctor
```

Comment:
include some validation in all your models. When you demo, creation of anything should appear meaning. Convey the message to Dipendra as well.